application.yml

```diff
@@ -54,6 +54,23 @@ management:
       prometheus:
         enabled: true
 
+# Chaos Engineering
+---
+spring:
+  config:
+    activate:
+      on-profile: chaos-monkey      
+management.endpoint.chaosmonkey.enabled: true
+chaos:
+  monkey:
+    enabled: true
+    watcher:
+      component: false
+      controller: false
+      repository: true
```

Comment:
Do you know why the repository watcher is set to true?

Reply:
That's a sensible default but we can set it to `false`

Reply:
Converted it to `false`